src/rules/matchDescription.js

```diff
@@ -47,18 +52,36 @@ export default iterateJsdoc(({
   tags.some((tag) => {
     const description = _.trimStart(tag.description, '- ');
 
-    return validateDescription(description, tag.tag);
+    return validateDescription(description, tag);
   });
 }, {
+  contextDefaults: true,
   meta: {
     schema: [
       {
         additionalProperties: false,
         properties: {
+          contexts: {
+            oneOf: [
+              {
+                items: {
+                  type: 'string'
+                },
+                type: 'array'
+              },
+              {
+                type: 'string'
+              }
+            ]
+          },
           matchDescription: {
             format: 'regex',
             type: 'string'
           },
+          noDefaults: {
```

Comment:
Why this option? Why not just setting `contexts`?

Reply:
It is extra work for users. I believe people will almost always want to keep the defaults. Also, that is the existing behavior for `require-description`.

Reply:
> Why this option? Isn't it more intuitive to let provided contexts to override the default context?

Agreed. Otherwise, I need to be aware of what the defaults are. `contexts` should just override the default.

Reply:
There have been a few rules already added using this approach, e.g., `require-description`, `check-types`. Revert them as well?

Reply:
(`check-types` is somewhat different in referring to avoid the defaults for your preferred types, while `require-description` is more like as in this PR.)

Reply:
I went ahead and added a commit to make a breaking change to remove `noDefaults` from `require-description` (and for `match-description`) though I haven't done so for `check-types` unless you want to confirm. Let me know if that is ok.

Reply:
@gajus : Since your approval kind of overlapped with my reversion, I just wanted to confirm--are you happy with my breaking change to undo `require-description`'s use of `noDefaults` (in addition to removing it in `match-description`)?